lib/fluent/plugin/out_datadog.rb

```diff
@@ -112,6 +113,9 @@ def write(chunk)
       if @include_tag_key
         record[@tag_key] = tag
       end
+      if @timestamp_key and time:
+        record[@timestamp_key] = time.utc.iso8601(3)
```

Comment:
Do we want to overwrite the field if it exists @NBParis? Or do we want to keep the existing one?

Reply:
Makes sense to keep an existing field if someone sets if explicitly, let me update the PR. Perhaps it could produce a warning, but I don't know if that's possible at all.

Reply:
PR updated.